Services/UI/OJS.Services.Ui.Models/Submissions/TestRunServiceModel.cs

```diff
@@ -24,8 +24,15 @@ public class TestRunServiceModel : IMapExplicitly
 
         public string? UserOutputFragment { get; set; }
 
+        public bool IsTrialTest { get; set; }
+
         public void RegisterMappings(IProfileExpression configuration)
             => configuration.CreateMap<TestRun, TestRunServiceModel>()
-                .ForMember(d => d.ResultType, opt => opt.MapFrom(s => s.ResultType.ToString()));
+                .ForMember(
+                    d => d.ResultType,
+                    opt => opt.MapFrom(s => s.ResultType.ToString()))
+                .ForMember(
+                    d => d.IsTrialTest,
+                    opt => opt.MapFrom(s => s.Test.IsTrialTest));
```

Comment:
Wouldn't be better to add `IsTrial` column in the TestRun itselft. We delete the testruns after each run, so we will not have incosistent data if the test is changed and is not longer trial...